packages/cli/uni-builder/tests/__snapshots__/postcssLegacy.test.ts.snap

```diff
@@ -143,55 +143,6 @@ exports[`plugin-postcssLegacy > should allow tools.postcss to override the plugi
           "postcssOptions": {
             "config": false,
             "plugins": [
-              {
-                "Once": [Function],
```

Comment:
Why are these snapshots been removed

Reply:
There was a problem with this case before. The user's `tools.postcss` object configuration should override the default plugins.